forte/processors/base/batch_processor.py

```diff
@@ -208,7 +208,7 @@ def _process_batch(
         pred = self.predict(batched_data)
         self.pack_all(packs, contexts, pred)
 
-    def predict(self, data_batch: Dict) -> Dict[str, List[Any]]:
+    def predict(self, data_batch: Dict) -> Dict[str, Dict[str, List[Any]]]:
```

Comment:
hmm, curious why this type is wrong before. could you help me understand how we can determine the type here?